### PR TITLE
Fixing WALA-VERIFY-HOSTNAME-CHANGE test

### PR DESCRIPTION
**Pull request by sharsonia**
This tets is failing for Ubuntu 18.0 and 20.04.

### CODE REVIEW

**Review by johnsongeorge-w — DISMISSED** [review]
Add comment for the change in condition.

**Comment by sharsonia** [comment]
image

File: Testscripts/Linux/WALA-VERIFY-HOSTNAME-CHANGE.py [hunk 1]
```diff
@@ -70,7 +70,8 @@ def ChangeHostName(expectedHost):
     expected_filter_string = "Detected hostname change: {0} -> {1}".format(expectedHost, changed_hostname)
     matchCount = Run("grep -i '"+expected_filter_string+"' /var/log/waagent.log | wc -l")
     RunLog.info('Get matchCount {0}'.format(matchCount))
-    if int(matchCount.rstrip()) == 1 and CheckHostName(changed_hostname) and int(fail_error_warn_count.rstrip()) == 0:
+    # Changed matchCount condition to be >= 1 since it showed up 4 time sin waagent.log file.
```
Comment (johnsongeorge-w) [inline Testscripts/Linux/WALA-VERIFY-HOSTNAME-CHANGE.py]: Typo in the comment ```time sin```

**Comment by LiliDeng** [comment]
it failed for Ubuntu 18.0 and 20.04 caused by not found ifdown and ifup commands on these distro, it is a known waagent issue.

**Review by squirrelsc — APPROVED** [review]

**Review by johnsongeorge-w — APPROVED** [review]

**Pull request merged**